### PR TITLE
DCS-1192: generate fake data for non-BaSM prison transfers

### PR DESCRIPTION
**Pull request by GurnankCheema**
Usage of fake data is a temporary measure. The api will eventually get the data from Nomis.

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementServiceTest.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          pncNumber = "1234/1234589A",
+          date = LocalDate.of(2021, 1, 2),
+          moveType = MoveType.PRISON_TRANSFER,
+          fromLocation = "MDI"
+        )
+      )
+
+    val moves = movementService.getMovements("MDI", LocalDate.of(2020, 1, 2))
+
+    Assertions.assertThat(moves).isEqualTo(
```
Comment (andrewrlee) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementServiceTest.kt:48]: often we would statically import Assertions so you wouldn't need to specify it here

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementService.kt [hunk 2]
```diff
@@ -5,6 +5,7 @@ import uk.gov.justice.digital.hmpps.welcometoprison.model.basm.BasmService
 import java.time.LocalDate
 
 @Service
-class MovementService(val basmService: BasmService) {
-  fun getMovements(agencyId: String, date: LocalDate) = basmService.getMoves(agencyId, date, date)
+class MovementService(val basmService: BasmService, val prisonService: PrisonService) {
+  fun getMovements(agencyId: String, date: LocalDate) =
+    basmService.getMoves(agencyId, date, date) + (prisonService.getMoves(agencyId, date))
```
Comment (andrewrlee) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementService.kt:10]: could remove the brackets

**Review by andrewrlee — COMMENTED** [review]
awesome, nice work!

File: src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementServiceTest.kt (hunk 1 above)
Comment (GurnankCheema) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementServiceTest.kt]: done

File: src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementService.kt (hunk 2 above)
Comment (GurnankCheema) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/MovementService.kt]: done

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**